README.md

```diff
@@ -1,54 +1,155 @@
+
+</div>
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <!-- <a href="https://github.com/metakgp/gyft">
+  </a> -->
+
+  <h1 align="center">kronos2.0</h1>
+  <br />
+    <a href="https://github.com/metakgp/gyft/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/metakgp/gyft/issues">Request Feature</a>
+  </p>
+</div>
+<!-- TABLE OF CONTENTS -->
+<details>
+<summary>Table of Contents</summary>
+
+- [About The Project](#about-the-project)
+- [Getting Started](#getting-started)
+  - [Without Docker](#without-docker)
+  - [using docker](#using-docker)
+- [Usage](#usage)
+- [Maintainer(s)](#maintainers)
+
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
 
 This webapp displays previous year's grade distribution. It has been hosted with streamlit as an app but is still under development.
 
 You can go [here](https://kronos.streamlit.app/) for the live version of the project. Make your choices wisely :)
 
 You can find the container image [here](https://hub.docker.com/repository/docker/spookbite/kronosv2).
 
-# Setup 
+# Getting started
 
 ## (Without Docker)
 
-### Install virtual environment module
-```shell
-pip install virtualenv
-```
 
-### Create a virtual environment (say env) 
+### 1. Install virtual environment module
 ```shell
-python -m venv env
+    pip install virtualenv
 ```
 
-### Activate the virtual environment 
+### 2. Create a virtual environment (say env) 
 ```shell
-source env/bin/activate
+    python -m venv env
 ```
 
-### Install Dependencies 
-```shell
-pip install -r requirements.txt
-```
+### 3. Activate the virtual environment 
+  ```shell
+      source env/bin/activate
+  ```
 
-Run the following code on your terminal:
-```shell
-streamlit run app.py
-```
+### 4. Install Dependencies 
+  ```shell
+      pip install -r requirements.txt
+  ```
+
+### 5. Run the following code on your terminal:
+  ```shell
+      streamlit run app.py
+  ```
 
 ## Using Docker
 
-### Pull the Docker image
+### 1. Pull the Docker image
 ```shell
-docker pull spookbite/kronosv2:latest
+    docker pull spookbite/kronosv2:latest
 ```
 
-### Run the container
+### 2. Run the container
 ```shell
-docker run -p 8501:8501 spookbite/kronosv2
+    docker run -p 8501:8501 spookbite/kronosv2
 ```
 
-### You can change the {PORT}:8501 to your desired PORT.
-
-## Credits
+You can change the {PORT}:8501 to your desired PORT.
+
+# Usage
+<!-- UPDATE -->
+Following is an example how this project works with streamlit.
+
+<div align="center">
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/37b7a1cf-0956-4545-9f31-6c45cc08b978">
+    </a>
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/f08c7a0a-1fd3-4c11-ab13-92bc120fea7f">
+
+  </a>
+</div>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+# Maintainer(s)
```

Comment:
https://github.com/metakgp/README.md/blob/256ba6d22b41617eb1b4223d12815260b038903d/README.md?plain=1#L154

Don't forget these one liner description for this and creator section as well.